pandas/tests/io/json/test_pandas.py

```diff
@@ -854,7 +854,7 @@ def test_date_format_frame(self, date, date_unit):
             json = df.to_json(date_format="iso")
         result = read_json(json)
         expected = df.copy()
-        # expected.index = expected.index.tz_localize("UTC")
+        expected.index = expected.index.tz_localize("UTC")
```

Comment:
so this fixes a bug?

Reply:
No this was intentionally commented out here https://github.com/pandas-dev/pandas/pull/27618/files#r311722007 just coming full circle on that now that date alignment is consistent in JSON whether an item appears in the index or in the values